packages/api/src/graphql/document/mutation/update-document.ts

```diff
@@ -23,41 +24,29 @@ const UPDATE_DOCUMENT = gql`
   }
 `;
 
-interface DocumentResponse {
-  witness: MerkleWitness;
-}
-
 export const updateDocument = async (
   databaseName: string,
   collectionName: string,
   documentQuery: JSON,
-  documentRecord: DocumentEncoded,
-  token: string
+  documentRecord: DocumentEncoded
 ): Promise<NetworkResult<MerkleWitness>> => {
   return handleRequest(async () => {
-    const { data, errors } = await client.mutate<{
-      documentUpdate: DocumentResponse;
-    }>({
+    const { data, errors } = await client.mutate({
       mutation: UPDATE_DOCUMENT,
       variables: {
         databaseName,
         collectionName,
         documentQuery,
         documentRecord,
-      },
-      context: {
-        headers: {
-          authorization: `Bearer ${token}`,
-        },
-      },
+      }
     });
 
     const response = data?.documentUpdate;
```

Comment:
Instead of question mark `?` in:
```ts
const response = data?.documentUpdate;
```
You can try:
```ts
const { data: {documentUpdate}, errors } = await client.mutate({
```